package/MDAnalysis/core/groups.py

```diff
@@ -907,7 +913,7 @@ def forces(self):
     def forces(self, values):
         ts = self._u.trajectory.ts
         try:
-            ts.forces[self._ix, :] = forces
+            ts.forces[self._ix, :] = values
```

Comment:
one would think so. I'm currently writing tests for the write function. If you find more spots where tests are needed to be triggered by this change please mark them